kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -164,18 +166,17 @@ class Transport(val options: TransportSettings) {
 
         // Request body
         val json = io.ktor.client.features.json.defaultSerializer()
-        // TODO make our request body form encoded
 
         val builder = HttpRequestBuilder()
         // Set the request method
         builder.method = method.value
 
         // Handle the headers
-        val agentTag = "LookerSDK Kotlin ${options.apiVersion}"
         val headers = options.headers.toMutableMap()
-        headers["User-Agent"] = agentTag
+        headers["User-Agent"] = AGENT_TAG
```

Comment:
I wasn't aware that we should leave the "User-Agent" tag in the headers now that we've added the "x-looker-appid" tag. I can change this in the python SDK if it's intended.

Reply:
It doesn't hurt to set `User-Agent` and it may end up preserved in some scenarios. `x-looker-appid` is something that should be preserved throughout the request lifecycle.